cmd/eksctl-anywhere/cmd/options.go

```diff
@@ -59,6 +64,7 @@ type clusterOptions struct {
 	fileName             string
 	bundlesOverride      string
 	managementKubeconfig string
+	installPackages      string
```

Comment:
This is going to be available to `delete cluster` commands - that seems wrong? Its also a little unclear why we need package installation on an abstract set of cluster options. This is only used in cluster creation so perhaps we leave it on `createClusterOptions` for now.

Reply:
The delete and upgrade commands previously both called directoriesToMount using the createClusterOptions variable (cc) initialized in createcluster.go. I added this so that each command could use it's own options but I think you're right and I ca work this out of the shared function - looking into that.

Reply:
Moved this out of the shared clusterOptions and back into `createClusterOptions`